data/presets/emergency/fire_service_inlet.json

```diff
@@ -0,0 +1,32 @@
+{
+    "icon": "temaki-fire_hydrant",
```

Comment:
What do you think about the `temaki-plumber`-icon?

Reply:
I'll leave it up to you and @tyrasd to decide, from my perspective the most important thing is that it's different to the fire hydrant icon



<details>
<summary>these are some other icons we should probably avoid since they're used by similar features:</summary>

![image](https://user-images.githubusercontent.com/16009897/223603634-3ae1819f-095d-4714-a3c8-8d25b1b04b6a.png)


</details>

Reply:
I decided me for the  `maki-water` icon.
![image](https://user-images.githubusercontent.com/73886169/224081526-b6958c0b-9fa7-463f-a5a7-68d178e68ef1.png)